app/controllers/users/notes_controller.rb

```diff
@@ -12,6 +12,13 @@ def index
     end
 
     def show
+      respond_to do |format|
+        format.html { respond_with note.author, note }
```

Comment:
Why we need `format.html { respond_with note.author, note }`?

Reply:
`ActionController::UnknownFormat` is raised when trying to view note without this line